tests/test_contamination_tech_reps.nf.test

```diff
@@ -19,7 +19,7 @@ nextflow_pipeline {
             assertAll(
                 { assert workflow.success },
                 { assert snapshot(UTILS.removeNextflowVersion("$outputDir")).match("software_versions") },
-                { assert workflow.trace.succeeded().size() == 73 },
+                { assert workflow.trace.succeeded().size() == 77 },
```

Comment:
Why is it having more tasks now?

Reply:
There have been several changes in the code that are responsible for this. The most important ones are:
1. The replacement of 1 local module "blat" (which had several functions) for 3 new nf-core modules: blat, gawk and seqkit.
2. In the changes, you'll see that I've changed a queue channel for a channel value. Before, only the first element of the queue was being process, as the channel was consumed. This change allows for all the elements to be processed.

Reply:
Makes sense. thanks!